### ISSUE TITLE
Hide inferred types when already given explicitly

### ISSUE THREAD

**Issue by ealap** [nushell/vscode-nushell-lang#183]
...or can we at least have an option to do it without hiding it for all other cases.

I personally get confused seeing double when I choose to explicitly annotate some variables.
![image](https://github.com/nushell/vscode-nushell-lang/assets/15620712/a2acd74c-533c-40b0-bb3c-34356f345bc5)

**Comment by fdncred** [comment]
Agreed. The Inlays shouldn't be shown if the type is explicitly provided. We'd accept a PR to make that change.

**Comment by Mrfiregem** [comment]
It looks like this was fixed at some point? Picture with v2.0.5. Thanks 👍 

<img width="275" height="122" alt="Image" src="https://github.com/user-attachments/assets/cc8e1249-a86e-44eb-a796-58a4dfaa44e3" />
